CHANGELOG.md

```diff
@@ -1,3 +1,114 @@
+### citus v9.5.11 (April 20, 2023) ###
+
+* Fixes a bug that could break `DROP SCHEMA/EXTENSON` commands when there is a
+  columnar table (#5458)
```

Comment:
A bizarre note: This is a PR that got merged in 2021, that is backported in 2022 and now being released in 2023.